### PR TITLE
Improve button style

### PR DESCRIPTION
**Pull request by kyu9341**
## Changes
- Improved button styling to match chatGPT's style.
  - button
- Improved code style and logic.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,47 +1,34 @@
-
 // Create the button and add an id
-var button = document.createElement("button");
+const button = document.createElement("button");
+
 button.innerHTML = "Save Chat";
-button.setAttribute("id", "chatGPT-save-button");
+button.setAttribute("id", "keeper-save-button");
```
Comment (wjrmffldrhrl) [inline script.js]:
```suggestion
button.setAttribute("id", "save-chat-button");
```

File: script.js [hunk 2]
```diff
@@ -1,47 +1,34 @@
...
 
 // Create a div to hold the button
-var buttonDiv = document.createElement("div");
-buttonDiv.setAttribute("id", "chatGPT-button");
+const buttonDiv = document.createElement("div");
+
+buttonDiv.setAttribute("id", "keeper-button-container");
```
Comment (wjrmffldrhrl) [inline script.js]:
```suggestion
buttonDiv.setAttribute("id", "save-chat-button-container");
```

File: script.js [hunk 3]
```diff
@@ -1,47 +1,34 @@
...
 buttonDiv.appendChild(button);
 
 // Add the button to the top right corner of the screen
 document.body.appendChild(buttonDiv);
 
 // Add event listener for button click
-document.getElementById("chatGPT-save-button").addEventListener("click", saveChat);
-
+document.getElementById("keeper-save-button").addEventListener("click", saveChat);
```
Comment (wjrmffldrhrl) [inline script.js]:
```suggestion
document.getElementById("save-chat-button").addEventListener("click", saveChat);
```

File: style.css [hunk 4]
```diff
@@ -1,20 +1,31 @@
...
-    display: inline-block;
-    font-size: 16px;
-    margin: 4px 2px;
-    cursor: pointer;
-  }
-  
\ No newline at end of file
+}
+
+#keeper-save-button {
```
Comment (wjrmffldrhrl) [inline style.css]:
```suggestion
#save-chat-button {
```

File: style.css [hunk 5]
```diff
@@ -1,20 +1,31 @@
...
+  display: inline-block;
+  font-size: 14px;
+  margin: 4px 2px;
+  border: none;
+  border-radius: 6px;
+  cursor: pointer;
+  min-width: max-content;
+}
+
+#keeper-save-button:hover {
```
Comment (wjrmffldrhrl) [inline style.css]:
```suggestion
#save-chat-button:hover {
```

File: style.css [hunk 6]
```diff
@@ -1,20 +1,31 @@
...
+  transition-duration: 80ms;
+  background-color: #32c59f;
+}
+
+#keeper-save-button:active {
```
Comment (wjrmffldrhrl) [inline style.css]:
```suggestion
#save-chat-button:active {
```

File: style.css [hunk 7]
```diff
@@ -1,20 +1,31 @@
 /* Add the button to the top right corner of the screen */
-#chatGPT-button {
+#keeper-button-container {
```
Comment (wjrmffldrhrl) [inline style.css]:
```suggestion
#save-chat-button-container {
```

**Comment by kyu9341** [comment]
Typically for libraries or extensions, there's a possibility that the CSS naming will conflict with the service, so we often prefix it with a prefix. here we've prefixed it with `keeper-`. What do you think?

**Comment by wjrmffldrhrl** [comment]
> Typically for libraries or extensions, there's a possibility that the CSS naming will conflict with the service, so we often prefix it with a prefix. here we've prefixed it with `keeper-`. What do you think?

Oh, I see  

How about `chatgpt-keeper-`?

**Comment by kyu9341** [comment]
> Oh, I see
> 
> How about `chatgpt-keeper-`?

The name feels a little long, but I think it's okay, I'll change it to that.

**Review by wjrmffldrhrl — APPROVED** [review]
LGTM

**Pull request merged**